packages/celer-code-generator/src/koroks.csv

```diff
@@ -641,36 +641,36 @@ R11,-1017.85,116.95,-1066.07,Basketball
 R12,-3109.25,34.17,-909.26,Confetti,Top of flagpole
 R13,-2669.55,375.56,-864.39,LiftRock,Top of waterfall
 R14,-2316.80,206.68,-806.45,LightChase
-R15,-2060.66,259.49,-775.78,RockCircle,3 on mushroom tree
```

Comment:
This is also comment for location but idk if it's needed

Reply:
Going to add something like it.